Formula/grafana.rb

```diff
@@ -1,8 +1,8 @@
 class Grafana < Formula
   desc "Gorgeous metric visualizations and dashboards for timeseries databases"
   homepage "https://grafana.com"
-  url "https://github.com/grafana/grafana/archive/v9.1.7.tar.gz"
-  sha256 "dac3d154901a9d5e0d1a21f3b3a5b06ea62e86aa03c4e2fa1fe38257887f3439"
+  url "https://github.com/grafana/grafana/archive/v9.1.8.tar.gz"
```

Comment:
why not 9.2.0?

Reply:
this is automatically created by release tool, we were releasing both grafana v9.1.8 and v9.2.0. should we keep the latest version instead?

Reply:
Yes, it should always be the latest version

Reply:
if it is, then the URL should point to v9.2.0 you are right, and we need to improve our release tool as well.